gui/next/src/lib/ui/Copy.svelte

```diff
@@ -0,0 +1,102 @@
+<!-- ================================================================== -->
+<script>
+
+// imports
+// ------------------------------------------------------------------------
+import Icon from '$lib/ui/Icon.svelte';
+
+
+// properties
+// ------------------------------------------------------------------------
+// what will be copied to clipboard (any type, converted to string)
+export let text;
+// if the text was already copied a second ago (bool)
+let copied = false;
+// if the copying is in progess, used for animation (bool)
+let copying = false;
+// if an error occured when trying to copy (bool)
+let error = false;
+
+
+// purpose:   copy text to clipboard
+// returns:   triggers the button animation
+// ------------------------------------------------------------------------
+const copy = () => {
```

Comment:
@z-x Instead of writing comments you could use different name for the function
`copyToClipboardAndTriggerAnimation`

This function does not return anything.